eslint.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": "eslint:recommended",
+    "env": {
+        "amd": true,
+        "browser": true
+    },
+    "globals": {
+        "jQuery": true,
+        "process": true,
+        "module": true,
+        "Buffer": true,
+        "Globalize": true
+    },
+    "rules":{
+        "no-constant-condition": 1,
+        "no-control-regex" : 0,
+        "no-empty": 1,
+        "no-mixed-spaces-and-tabs": 1,
+        "no-redeclare": 0,
+        "no-shadow-restricted-names": 1,
+        "no-unexpected-multiline": 1,
+        "no-useless-escape": 1
+    }
+}
```

Comment:
Please add an empty new line at the end of the file.

Reply:
added.